src/Microsoft.Identity.Client/ITokenCache.cs

```diff
@@ -64,47 +65,103 @@ public interface ITokenCache
         void SetBeforeWrite(TokenCacheCallback beforeWrite);
 
         /// <summary>
-        /// Unified Only
+        /// Serializes the token cache to the MSAL.NET 3.x unified cache format, which is compatible with other
+        /// MSAL desktop versions (MSAL.Python and MSAL.Java for instance).
+        /// If you need to maintain SSO between an application using MSAL 2.x and this application using MSAL 3.x,
+        /// you might also want to serialize with Serialize and Deserialize, otherwise just use
+        /// SerializeMsalV3 and DeserializeMsalV3.
         /// </summary>
-        /// <returns></returns>
-        byte[] Serialize();
+        /// <returns>Byte stream representation of the cache</returns>
+        byte[] SerializeMsalV3();
 
         /// <summary>
-        /// 
+        /// Deserializes the token cache to the MSAL.NET 3.x unified cache format, which is compatible with other
+        /// MSAL desktop versions (MSAL.Python and MSAL.Java for instance).
+        /// If you need to maintain SSO between an application using MSAL 2.x and this application using MSAL 3.x,
+        /// you might also want to serialize with Serialize and Deserialize, otherwise just use
+        /// SerializeMsalV3 and DeserializeMsalV3
         /// </summary>
-        /// <param name="unifiedState"></param>
-        void Deserialize(byte[] unifiedState);
+        /// <param name="bytes">Byte stream representation of the cache</param>
+        /// <param name="merge">Future: Boolean enabling merging the cache into the existing cache.</param>
+        /// <remarks>
+        /// Merge is not yet available.
```

Comment:
(merge=true) if you try to use it this will throw a NotImplementedException.
This will be available in a later version of MSAL.NET